news_layout/css/main.css

```diff
@@ -1 +1,174 @@
-/* Write your CSS here */
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+body {
+  background-color: #FDEDEA;
+  color: #3E3E54;
+  font-family: 'Roboto Condensed', sans-serif;
```

Comment:
You've already set this font-family to everything so you don't need to specify it again unless it changes from Roboto Condensed.